resources/assets/lib/chat/chat-state-store.ts

```diff
@@ -157,6 +160,19 @@ export default class ChatStateStore implements DispatchListener {
     this.selectChannel(event.channelId);
   }
 
+  @action
+  private handleFriendUpdated(event: FriendUpdated) {
+    if (!this.isChatMounted) return;
+
+    // TODO: update other tabs...somehow
+    const channel = this.channelStore.groupedChannels.PM.find((value) => value.pmTarget === event.userId);
+    if (channel != null) {
+      getChannel(channel?.channelId).done((response) => {
+        channel.updateWithJson(response.channel);
+      });
+    }
```

Comment:
make a `refresh` function in `Channel` class and call it here? (`channel?.refresh()`)